docs/manual/install/index.rst

```diff
@@ -111,3 +111,32 @@ Stable versions of Qtile can be installed from PyPI:
 As long as the necessary libraries are in place, this can be done at any point,
 however, it is recommended that you first install xcffib to ensure the
 cairo-xcb bindings are built (see above).
+
+Wayland
+-------
+
+Qtile can also be run as a Wayland compositor rather than an X11 window
+manager. This does not require an X server or ``xcffib`` to be installed.
+Instead Qtile uses pywlroots_, a Python binding around the wlroots_ library,
+pywayland_ and python-xkbcommon_. First, wlroots needs to be installed, and
```

Comment:
We could add a `wayland` dependency install in `options.extras_require` in `setup.cfg`. Really, I think `pywlroots` will pull in `pywayland` and `python-xkbcommon`, so for now that should be the only dependency we need to add there.